sumOfOther.js

```diff
@@ -0,0 +1,10 @@
+function sumOfOther(arr) {
+  let newArr = [];
+  let buf;
+  let red = arr.reduce(function(x,y) { return x + y; });
+  for (let i = 0, n = arr.length; i < n; i++){
```

Comment:
@GoldenKit n = arr.length is not necessary.
Missing space before opening brace.